### PR TITLE
Feature: option in change custom notice text

### PR DESCRIPTION
**Pull request by anisAronno**

### CODE REVIEW

**Comment by tareq1988** [comment]
You've changed the coding convention from WordPress standard to PSR standard, which is not ideal for this library and has been skipped from the beginning. So revert all the changes and commit again with fixes.

**Comment by tareq1988** [comment]
You've changed the coding convention from WordPress standard to PSR standard, which is not ideal for this library and has been skipped from the beginning. So revert all the changes and commit again with fixes.

**Comment by saiftheboss7** [comment]
@anis3139  fix and give PR again

**Comment by anis3139** [comment]
> You've changed the coding convention from WordPress standard to PSR standard, which is not ideal for this library and has been skipped from the beginning. So revert all the changes and commit again with fixes.

I hope solved WordPress standard coding issue. Please check..

File: src/Insights.php [hunk 1]
```diff
@@ -104,8 +104,8 @@ public function add_extra( $data = array() ) {
      *
      * @return \self
      */
-    public function notice( $text ) {
-        $this->notice = $text;
+    public function notice(string $text='' ) {
+        $this->notice = sanitize_text_field($text);
```
Comment (tareq1988) [inline src/Insights.php:108]: It's not necessary to sanitize text here as it's not coming from the general untrusted users. Only when we are outputting, we need to escape the output with [escape functions](https://developer.wordpress.org/themes/theme-security/data-sanitization-escaping/#escaping-securing-output).

**Review by tareq1988 — CHANGES_REQUESTED** [review]

File: src/Insights.php [hunk 2]
```diff
@@ -104,7 +104,7 @@ public function add_extra( $data = array() ) {
      *
      * @return \self
      */
-    public function notice( $text ) {
+    public function notice(string $text='' ) {
```
Comment (tareq1988) [inline src/Insights.php:107]: Adding type-hinting here will require the user's to use PHP 7.0+. Remove it.

**Review by tareq1988 — CHANGES_REQUESTED** [review]

**Pull request merged**